OrbitCore/Log.h

```diff
@@ -10,10 +10,6 @@
 
 #include "Threading.h"
 
-#define ORBIT_LOG(msg) GLogger.Log(OrbitLog::Global, msg)
-#define ORBIT_LOGV(var) GLogger.Log(OrbitLog::Global, #var, var)
-#define ORBIT_VIZV(var) GLogger.Log(OrbitLog::Viz, #var, var)
-
 class OrbitLog {
```

Comment:
Do we use `OrbitLog` and `Logger` classes after this change?

Reply:
Sorry, just noticed that you removed it in the next PR

Reply:
#1007